data/extension/init/templates/rust/envoy.access_loggers/default/src/logger.rs

```diff
@@ -12,28 +10,25 @@ use super::stats::SampleAccessLoggerStats;
 pub struct SampleAccessLogger<'a> {
     config: SampleAccessLoggerConfig,
     stats: SampleAccessLoggerStats,
-    // This example shows how to use Time API provided by Envoy host.
```

Comment:
this lied as it only showed how to initialize it. the rust linter found this was never used